### PR TITLE
[qa] Update openwisp-utils-qa-checks migation check

### PR DESCRIPTION
**Pull request by waleko**
Update `openwisp-utils-qa-checks` (related to openwisp/openwisp-utils#63 for this repo)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,9 +19,11 @@ env:
    - SAMPLE_APP=1  # sample app tests
 
 before_install:
+  # Temporary, remove when openwisp-utils 0.4 is released
+  - pip install -e git+git://github.com/openwisp/openwisp-utils#egg=openwisp_utils[qa]
```
Comment (atb00ker) [inline .travis.yml]: Can you please let me know why openwisp-utils 0.3 wouldn't work here? :smile:

Comment (atb00ker) [reply .travis.yml]: Nevermind, got it! :smile:

**Comment by waleko** [comment]
I fixed most of the stuff, but `checkmigrations` still fails with sample app. Previously, there was just an easy workaround:
https://github.com/openwisp/django-ipam/blob/b4db5413dfe88262cac7d51ebaa210e9b9990480/.travis.yml#L34
Perhaps, we can make an if to skip checkmigrations if `SAMPLE_APP=1`, but maybe it's a bit of a redundancy.

**Review by atb00ker — COMMENTED** [review]
Can't we try the easy fix again? :smile:

```
if [[ $TRAVIS_PYTHON_VERSION >= 3.5 && $SAMPLE_APP != "1" ]];
    openwisp-utils-qa-checks --migration-path "./django_ipam/migrations/ ./tests/sample_ipam/migrations/"
    --migration-module django_ipam
fi
```

**Review by atb00ker — APPROVED** [review]

File: .travis.yml [hunk 2]
```diff
@@ -20,24 +20,21 @@ env:
...
+script:
+  - >
+    openwisp-utils-qa-checks --migration-path "./django_ipam/migrations/ ./tests/sample_ipam/migrations/"
+    --skip-checkmakemigrations
   - |
-    if [[ $TRAVIS_PYTHON_VERSION == 3.5 && $SAMPLE_APP != "1" ]]; then
-      OUTPUT=$(./tests/manage.py makemigrations --dry-run django_ipam)
-      echo $OUTPUT
-      echo $OUTPUT | grep "No changes detected"
+    if [[ $TRAVIS_PYTHON_VERSION >= 3.5 && $SAMPLE_APP != "1" ]]; then
```
Comment (atb00ker) [inline .travis.yml]: I just saw travis failing:
I think:
```
TRAVIS_PYTHON_VERSION >= 
```
is not allowed, can you please try `==` to fix it ? 
Use python version 3.6 ! :smile:

File: .travis.yml [hunk 3]
```diff
@@ -20,24 +20,21 @@ env:
...
+script:
+  - >
+    openwisp-utils-qa-checks --migration-path "./django_ipam/migrations/ ./tests/sample_ipam/migrations/"
+    --skip-checkmakemigrations
   - |
     if [[ $TRAVIS_PYTHON_VERSION == 3.5 && $SAMPLE_APP != "1" ]]; then
-      OUTPUT=$(./tests/manage.py makemigrations --dry-run django_ipam)
-      echo $OUTPUT
-      echo $OUTPUT | grep "No changes detected"
+      openwisp-utils-qa-checks --skip-isort --skip-flake8 --skip-checkmigrations --skip-checkendline --skip-checkcommit --migration-module django_ipam
```
Comment (nemesifier) [inline .travis.yml]: please use multiline statements for better readability

**Review by nemesifier — CHANGES_REQUESTED** [review]

**Review by nemesifier — APPROVED** [review]

**Pull request merged**